serialize.go

```diff
@@ -533,6 +555,8 @@ func DeserializeFragmentInfo(fragmentInfo FragmentInfo, buffer *Buffer, arrayURI
 	defer C.free(unsafe.Pointer(cArrayURI))
 
 	ret := C.tiledb_deserialize_fragment_info(fragmentInfo.context.tiledbContext, buffer.tiledbBuffer, C.tiledb_serialization_type_t(serializationType), cArrayURI, cClientSide, fragmentInfo.tiledbFragmentInfo)
+	runtime.KeepAlive(fragmentInfo)
```

Comment:
This `FragmentInfo` (and the one in line 581) is passed by value, so this call does not do anything. We might want to change it now or in the future to take a pointer, don't know if the break is tolerable.